### PR TITLE
Create a nuget package also on windows to fix compilation

### PR DESCRIPTION
**Pull request by vmartos**
Windows nuget just adds a framework assembly dependency
since is not posible to create empty nuget packages

### CODE REVIEW

File: recipes/sparkle/sparkle.recipe [hunk 1]
```diff
@@ -19,35 +19,54 @@ class Recipe(recipe.Recipe):
...
+            shell.call('xcodebuild -configuration Release -target Sparkle',
+                    os.path.join(self.build_dir, 'Sparkle/3rdparty/Sparkle'))
+            shell.call("xbuild Sparkle.sln /p:Configuration=Release", self.build_dir)
+
+            shutil.copy(self.relative_path ('sparkle.nuspec.tpl'),
+                    nuspec)
+            
+        elif self.config.platform == Platform.WINDOWS:
+            shutil.copy(self.relative_path ('sparkle.nuspec.win.tpl'),
+                    nuspec)
```
Comment (vguzmanp) [inline recipes/sparkle/sparkle.recipe]: Better move it outside the if, setting a variable like `nuspec_tpl`

Comment (vguzmanp) [inline recipes/sparkle/sparkle.recipe]: Also, it looks better in one line

File: recipes/sparkle/sparkle.recipe [hunk 2]
```diff
@@ -19,35 +19,54 @@ class Recipe(recipe.Recipe):
...
+    def fetch(self):
+        if self.config.platform == Platform.DARWIN:
+            super(Recipe, self).fetch()
+
+    def extract(self):
+        if self.config.platform == Platform.DARWIN:
+            super(Recipe, self).extract()
+        elif self.config.platform == Platform.WINDOWS:
+            if not os.path.exists(self.build_dir):
+                os.mkdir(self.build_dir)
```
Comment (vguzmanp) [inline recipes/sparkle/sparkle.recipe]: do we only want to mkdir the build dir on windows?

Comment (vmartos) [inline recipes/sparkle/sparkle.recipe]: I have changed to mkdir also on linux!

**Pull request merged**